### PR TITLE
boot_to_desktop: add additional 60 seconds for Leap migration

### PR DESCRIPTION
**Pull request by nilxam**
Leap migration from 15.2 to 15.3 takes more time on rebooting to bootloader https://openqa.opensuse.org/tests/1725217#step/boot_to_desktop/2 , added additional timeout on `ZDUP` scenario.

- Verification run: https://openqa.opensuse.org/tests/1725427

### CODE REVIEW

File: tests/boot/boot_to_desktop.pm [hunk 1]
```diff
@@ -35,6 +35,8 @@ sub run {
     # needs additional time too.
     $timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION');
     $timeout += 60 if get_var('ENCRYPT');
+    # Add additional 60 seconds if the test suite is migration to newer Leap from older Leap
```
Comment (okurz) [inline tests/boot/boot_to_desktop.pm:38]: Please make the comment explain *why* Leap should be treated differently

**Review by okurz — CHANGES_REQUESTED** [review]

File: tests/boot/boot_to_desktop.pm (hunk 1 above)
Comment (nilxam) [reply tests/boot/boot_to_desktop.pm:38]: it just takes longer than the default timeout... `2009 packages to upgrade, 195 to downgrade, 175 new, 20 to remove` looks normal to me. or you're asking me to look performance issue?

Comment (okurz) [reply tests/boot/boot_to_desktop.pm:38]: Well, there is already `$timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION');` Maybe just extend that to include `ZDUP`? I don't see why Leap should take longer than Tumbleweed or SLE to boot after a zdup upgrade so this is why I asked for the comment to explain better

Comment (nilxam) [reply tests/boot/boot_to_desktop.pm]: ok then. updated.

**Review by jouyingbin — COMMENTED** [review]
LGTM. Thanks for the patch.

**Pull request merged**